src/app/task/task.component.ts

```diff
@@ -1,11 +1,12 @@
-import { Component, Input, OnInit } from "@angular/core";
+import { Component, Input, ViewEncapsulation } from "@angular/core";
 
 import { TaskDTO } from "../shared/TaskDTO";
 
 @Component({
 	selector: "task",
 	templateUrl: "./task.component.html",
-	styleUrls: ["./task.component.css"]
+	styleUrls: ["./task.component.css"],
+	encapsulation: ViewEncapsulation.None
```

Comment:
do you need this?

Reply:
yes, otherwise I can't access the style properties for p-card header's.
more specific, this: `.strikethrough .p-card .p-card-title {
	text-decoration: line-through;
}` has no effect unless I add that encapsulation property

Reply:
I tried other things too but couldn't find a better solution